src/sb_rand.c

```diff
@@ -343,6 +343,32 @@ void sb_rand_str(const char *fmt, char *buf)
   }
 }
 
+/*
+  Generates a random string of ASCII characters between '0' and 'z' of a length
+  between min and max. buf should have enough room for max len bytes. Returns
+  the number of characters written into the buffer.
+ */
+
+uint32_t sb_rand_varstr(char *buf, uint32_t min_len, uint32_t max_len)
+{
+  unsigned int i;
+  uint32_t num_chars;
+  if (max_len == 0) {
+    return; /* we can't be sure buf is long enough to populate, so be safe */
```

Comment:
This results in a "non-void function 'sb_rand_varstr' should return a value" warning in GCC builds and an error in Clang ones.

Reply:
oops, yeah, sorry about that. originally wasn't returning the number of bytes written and forgot to update this when I changed it. will fix shortly.